server/src/main/java/io/unitycatalog/server/persist/utils/RepositoryUtils.java

```diff
@@ -80,4 +80,20 @@ public static CatalogInfoDAO getCatalogByIdOrThrow(Session session, UUID catalog
     }
     return catalogInfoDAO;
   }
+
+  public static Pair<String, String> getCatalogAndSchemaNames(Session session, UUID schemaId) {
```

Comment:
Nit: Seems this method is useless for this PR, then I think we can remove it in this PR.